src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/jpa/repository/PrisonIncentiveLevelRepositoryTest.kt

```diff
@@ -240,4 +241,34 @@ class PrisonIncentiveLevelRepositoryTest : TestBase() {
     val missing = repository.findFirstByPrisonIdAndActiveIsTrueAndDefaultIsTrue("LEI")
     assertThat(missing).isNull()
   }
+
+  @Test
+  fun `find prisons that have active levels`(): Unit = runBlocking {
+    generateDefaultData()
+    // Generate inactive prisons that will not be returned:
+    listOf("BAS", "STD", "ENH", "EN2", "ENT").forEach { levelCode ->
+      listOf("EXI", "LEI").forEach { prisonId ->
+        val entity = PrisonIncentiveLevel(
+          levelCode = levelCode,
+          prisonId = prisonId,
+          active = false,
+          defaultOnAdmission = levelCode == "STD",
+
+          remandTransferLimitInPence = 6050,
+          remandSpendLimitInPence = 60500,
+          convictedTransferLimitInPence = 1980,
+          convictedSpendLimitInPence = 19800,
+
+          visitOrders = 2,
+          privilegedVisitOrders = 1,
+
+          new = true,
+        )
```

Comment:
You could your fancy `makePrisonIncentiveLevel()` helper and then set `active=false` and that would be more intent revealing/simpler (assuming I'm getting what you're doing here right, e.g. you're setting all levels in EXI/LEI prisons to inactive)